Cargo.toml

```diff
@@ -12,16 +12,29 @@ members = [
 
 [workspace.package]
 edition = "2021"
-version = "0.1.0"
-
-[workspace.dependencies]
-wgpu = "0.15"
+version = "0.0.1"
+license = "MIT/Apache-2.0"
+# homepage = "https://vello.dev" - Domain owned by us, but unused at present
+# rust-version = 
+repository = "https://github.com/linebender/vello"
 
 [package]
 name = "vello"
-version = "0.1.0"
-license = "MIT/Apache-2.0"
-edition = "2021"
+description = "An experimental GPU compute-centric 2D renderer"
+categories = ["rendering", "graphics"]
+keywords = ["2d", "vector-graphics"]                            # , "xilem"?
```

Comment:
I don't think "xilem" is an appropriate keyword. Otherwise, all this is fine.

Reply:
My logic was that it's the "backend" for Xilem, and it seemed useful to have all our "new stack" crates unified under one keyword umbrella

I'll remove this comment though; I'm not married to the idea.